src/redux/dialogsReducer.js

```diff
@@ -0,0 +1,24 @@
+import ACTION_TYPES from "./actionTypes";
+
+const dialogsReducer = (state, action) => {
+    switch(action.type) {
+        case ACTION_TYPES.UPDATE_NEW_MESSAGE_BODY: 
+        state.newMessageBody = action.payload;
+        return state;
+
+case ACTION_TYPES.SEND_MESSAGE: 
```

Comment:
There's something wrong with indentation - it's hard to read.

Reply:
done